osu.Game/Overlays/Settings/Sections/Input/KeyBindingRow.cs

```diff
@@ -601,6 +633,15 @@ protected override void Dispose(bool isDisposing)
                 if (keyCombinationProvider != null)
                     keyCombinationProvider.KeymapChanged -= updateKeyCombinationText;
             }
+
+            private bool keyCombinationExists(KeyCombination keyCombination)
+            {
+                var bindings = realm.Run(r => r.All<RealmKeyBinding>()
+                                               .Where(b => b.RulesetName == KeyBinding.RulesetName && b.Variant == KeyBinding.Variant)
+                                               .Detach());
```

Comment:
Detach should not be required as you aren't returning anything. In fact I'm not sure the way you're doing this is efficient – can you not move the `Contains` to inside the realm query rather than fetching first then performing the `Contains`?